localkube.go

```diff
@@ -15,20 +16,28 @@ func (lk *LocalKube) Add(server Server) {
 	lk.Servers = append(lk.Servers, server)
 }
 
-func (lk *LocalKube) Run(args []string, out io.Writer) error {
-	if len(args) < 2 {
+var containerized = flag.Bool("containerized", true, "Whether localkube is inside a container or not")
```

Comment:
Looks like inspecting /proc/1/cgroup should work, but I'd kind of prefer to keep this explicit to match the behavior of the kubelet itself.

Thoughts?